regexes/client/browsers.yml

```diff
@@ -5,6 +5,11 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# Covenant Eyes (https://www.covenanteyes.com/)
+- regex: 'Covenant%20Eyes/(\d+[\.\d]+)'
+  name: 'Covenant Eyes'
+  version: '$1'
+
```

Comment:
It's not a browser, it's a mobile app
https://apps.apple.com/au/app/covenant-eyes/id335318146?l=en